config/application.rb

```diff
@@ -114,7 +114,7 @@ class Application < Rails::Application
     config.middleware.insert_before 0, Rack::Cors do
       allow do
         origins do |source, _env|
-          next if source == IdentityConfig.store.domain_name
+          next if source == "https://#{IdentityConfig.store.domain_name}"
```

Comment:
Should we be considering `ENV['HTTPS']` for the scheme? Not sure if we need to support `http://`

Reply:
This would only be for our internal stuff, which should always be `https://`